FindMe/Parse Manager/DatabaseManager.m

```diff
@@ -0,0 +1,86 @@
+//
+//  DatabaseManager.m
+//  FindMe
+//
+//  Created by Nicholas Wayoe on 7/21/20.
+//  Copyright © 2020 Nicholas Wayoe. All rights reserved.
+//
+
+#import "DatabaseManager.h"
+#import "Parse/Parse.h"
+
+@implementation DatabaseManager
+
++ (void)createUser:(NSDictionary *)userDetails withCompletion:(void(^)(NSError *error))completion
+{
+    PFUser *user = [PFUser new];
+    user.username = userDetails[@"username"];
+    user.password = userDetails[@"password"];
+    user[@"firstName"] = userDetails[@"firstName"];
+    user[@"lastName"] = userDetails[@"lastName"];
+    if (userDetails[@"profilePhotoData"]) {
+        user[@"profilePhoto"] = [PFFileObject fileObjectWithData:userDetails[@"profilePhotoData"]];
+    }
+    else {
+        
+    }
+    [user signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * error) {
+        if (error) {
+            completion(error);
+        }
+        else
+            completion(nil);
+    }];
+}
+
++ (void)getUser:(void(^)(PFUser * user))completion
```

Comment:
Should this return a `User` or a `PFUser` ? Translating between these two is pretty key.